src/helpers/appBar.js

```diff
@@ -149,136 +151,147 @@ const CodeLabzAppBar = () => {
   if (authed) {
     return (
       <div className={classes.grow} data-testId="navbarloggedIn">
-        <AppBar position="static" color="white">
-          <Toolbar className={classes.toolbar}>
-            <Typography
-              className={classes.title}
-              variant="h6"
-              noWrap
-              data-testId="navbarBrand"
-            >
-              <Link to={"/"}>
-                <BrandName />
-              </Link>
-            </Typography>
+        <MainNavbar />
+      </div>
+    );
+    // return (
```

Comment:
why is the lines are commented? any reason?

Reply:
Can you move it to another file so that our code keep clean. Leaving a large chunk of commented code is not nice noh.

Reply:
Okay, Got it suree

Reply:
Done!